src/redisearch.h

```diff
@@ -250,24 +250,17 @@ typedef struct {
   double value;
 } RSNumericRecord;
 
-/* A vector record represents a vector similarity search result which has a specific *distance* from the
- * query vector in the vector index, and associated with *scoreField* */
-typedef struct {
-  double distance;
-  char *scoreField;
-} RSDistanceRecord;
-
 typedef enum {
   RSResultType_Union = 0x1,
   RSResultType_Intersection = 0x2,
   RSResultType_Term = 0x4,
   RSResultType_Virtual = 0x8,
   RSResultType_Numeric = 0x10,
-  RSResultType_Distance = 0x20,
-  RSResultType_HybridDistance = 0x40,
+  RSResultType_Metric = 0x20,
+  RSResultType_HybridMetric = 0x40,
```

Comment:
what is the difference?

Reply:
One is holding the metric value alone, the other is of type aggregate (used for hybrid queries), and has two children - 
1. `RSIndexResult` of type RSResultType_Metric that holds the metric.
2. `RSIndexResult` corresponding to the hybrid iterator's child's iterator type